docs/library/usage/self-hosting.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_label: "Self-hosting"
+---
+
+# Self-hosting the library
+
+:::caution
+
+Self-hosting Gruntwork Library is only available with a Gruntwork Enterprise subscription.
+
+:::
+
+We recommend accessing Gruntwork Library directly through our private repos hosted at https://github.com/gruntwork-io. However if your company requires that all Terraform modules are hosted internally, we offer a tool called [repo-copier](https://github.com/gruntwork-io/repo-copier) to support this use case.
```

Comment:
```suggestion
Self-hosting the Gruntwork Library is only available with a Gruntwork Enterprise subscription.

:::

We recommend accessing the Gruntwork Library directly through our private repos hosted at https://github.com/gruntwork-io. However if your company requires that all Terraform modules are hosted internally, we offer a tool called [repo-copier](https://github.com/gruntwork-io/repo-copier) to support this use case.
```

Reply:
I actually prefer it without the "the" because I think of the product name as "Gruntwork Library" just like a product name lik Heroku, Render.com or Supabase.

In that way of thinking you wouldn't refer to "self-hosting the Heroku." I think "the Gruntwork Library" only sounds ok because you're describing a library whose name is "Gruntwork."

Reply:
Sounds good.